cartographer/mapping/internal/2d/pose_graph/optimization_problem_2d.cc

```diff
@@ -325,7 +341,8 @@ std::unique_ptr<transform::Rigid3d> OptimizationProblem2D::InterpolateOdometry(
           .transform);
 }
 
-transform::Rigid3d OptimizationProblem2D::ComputeRelativePose(
+std::unique_ptr<transform::Rigid3d>
```

Comment:
The return type could be common::optional but I think consistency with the other  function has priority here.

Reply:
How you like.